### PR TITLE
Sqreen integration

### PR DESCRIPTION
**Pull request by arnaudbreton**
### What does this PR do?

The Sqreen integration enables you and your team to monitor your service security activity right from Datadog.

### Motivation

Sqreen acquisition by Datadog

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [X] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

N/A

### CODE REVIEW

File: sqreen/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Sqreen Integration
+
+## Overview
+
+The Sqreen integration enables you and your team to monitor your service security activity right from Datadog.
+
+It provides you with dashboards, security rules, log facets and saved views.
```
Comment (apigirl) [inline sqreen/README.md:7]:
```suggestion
It provides you with dashboards, security rules, log facets, and saved views.
```

File: sqreen/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+This integration is only available to Datadog AppSec design partners.
```
Comment (apigirl) [inline sqreen/README.md:9]: Is there somewhere to link that describes what an AppSec design partner is?

File: sqreen/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+![Sqreen Insights in Datadog][1]
+
+### Security Rules
+
+This integration contains 2 security rules: 
+* Security Incident detected by Sqreen
+* A user logged in from a new country
+
+Once setup, you'll be able to review their full definition from the [Security Rules list](https://app.datadoghq.com/security/configuration/rules?sort=rule&query=source%3Asqreen).
```
Comment (apigirl) [inline sqreen/README.md:21]:
```suggestion
Once set up, you'll be able to review their full definition from the [Security Rules list](https://app.datadoghq.com/security/configuration/rules?sort=rule&query=source%3Asqreen).
```

File: sqreen/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Setup
+
+### Configure the Sqreen -> Datadog data pipeline
+
+Sqreen streams your services security activity to Datadog as logs.
+
+Here's the steps to follow to set it up:
+
+1. Go to the [Sqreen Dashboard > Organization settings > Integrations](https://my.sqreen.com/profile/organization/integrations),
```
Comment (apigirl) [inline sqreen/README.md:31]:
```suggestion
1. Go to the [Sqreen Dashboard > Organization settings > Integrations](https://my.sqreen.com/profile/organization/integrations).
```

File: sqreen/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+2. Add a new "Push API" integration and select "datadog" as the destination,
```
Comment (apigirl) [inline sqreen/README.md:32]:
```suggestion
2. Add a new "Push API" integration and select `datadog` as the destination.
```

File: sqreen/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+3. Choose the corresponding intake server URL. For Datadog US: http-intake.logs.datadoghq.com. For Datadog EU: http-intake.logs.datadoghq.eu,
```
Comment (apigirl) [inline sqreen/README.md:33]:
```suggestion
3. Choose the corresponding intake server URL. For Datadog US: `http-intake.logs.datadoghq.com`. For Datadog EU: `http-intake.logs.datadoghq.eu`.
```

File: sqreen/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+4. In the "dd_api_key", provide a Datadog API key used to stream the Sqreen data as logs. [Datadog API keys management page](https://app.datadoghq.com/account/settings#api).
```
Comment (apigirl) [inline sqreen/README.md:34]:
```suggestion
4. In the `dd_api_key` field, provide a [Datadog API key][2] used to stream the Sqreen data as logs.
```

File: sqreen/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+### Troubleshooting
+
+In case you face any issues using this integration, please reach out to [support@sqreen.com](mailto:support@sqreen.com).
+
+## Payload Reference Documentation
+
+The log format is fully documented on the [Sqreen documentation](https://docs.sqreen.com/integrations/datadog-integration/).
+
+[1]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/sqreen/images/sqreen_dashboard.png
```
Comment (apigirl) [inline sqreen/README.md:48]:
```suggestion
[1]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/sqreen/images/sqreen_dashboard.png
[2]: https://app.datadoghq.com/account/settings#api 
[3]: https://docs.sqreen.com/integrations/datadog-integration/
```

File: sqreen/README.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+3. Choose the corresponding intake server URL. For Datadog US: http-intake.logs.datadoghq.com. For Datadog EU: http-intake.logs.datadoghq.eu,
+4. In the "dd_api_key", provide a Datadog API key used to stream the Sqreen data as logs. [Datadog API keys management page](https://app.datadoghq.com/account/settings#api).
+
+After a few seconds, the first logs with `source:sqreen` should be flowing in. 
+
+The assets bundled in this integration will be deployed into your Datadog account.
+
+### Troubleshooting
+
+In case you face any issues using this integration, please reach out to [support@sqreen.com](mailto:support@sqreen.com).
```
Comment (apigirl) [inline sqreen/README.md:42]:
```suggestion
For support, reach out to [support@sqreen.com](mailto:support@sqreen.com).
```

File: sqreen/README.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+## Payload Reference Documentation
+
+The log format is fully documented on the [Sqreen documentation](https://docs.sqreen.com/integrations/datadog-integration/).
+
```
Comment (apigirl) [inline sqreen/README.md:47]: You are missing the required Data collected section. For a good example, see https://github.com/DataDog/integrations-extras/tree/master/postman#data-collected

**Review by apigirl — COMMENTED** [review]
initial docs review

File: sqreen/README.md (hunk 2 above)
Comment (arnaudbreton) [reply sqreen/README.md:9]: @apigirl this integration will be fully private, meaning no accounts besides the design partners one should see it on the docs nor on the Dashboard. Knowing this, I'm not sure it's worth describing it. Wdyt?

Comment (anshumgargdd) [reply sqreen/README.md:9]: let's just say that it's only available to Datadog and Sqreen customers?

File: sqreen/README.md (hunk 10 above)
Comment (arnaudbreton) [reply sqreen/README.md:47]: Fixed

File: sqreen/README.md (hunk 2 above)
Comment (arnaudbreton) [reply sqreen/README.md]: Fixed

File: sqreen/README.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+![Sqreen Insights in Datadog][1]
+
+### Security Rules
+
+This integration contains 2 security rules: 
+* Security Incident detected by Sqreen
+* A user logged in from a new country
+
+Once set up, you'll be able to review their full definition from the [Security Rules list](https://app.datadoghq.com/security/configuration/rules?sort=rule&query=source%3Asqreen).
```
Comment (cswatt) [inline sqreen/README.md:21]:
```suggestion
Once set up, you'll be able to review these rules' full definitions from the [Security Rules list](https://app.datadoghq.com/security/configuration/rules?sort=rule&query=source%3Asqreen).
```

File: sqreen/README.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+## Setup
+
+### Configure the Sqreen -> Datadog data pipeline
+
+Sqreen streams your services security activity to Datadog as logs.
+
+Here's the steps to follow to set it up:
```
Comment (cswatt) [inline sqreen/README.md:29]:
```suggestion
To set it up:
```

File: sqreen/README.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Go to the [Sqreen Dashboard > Organization settings > Integrations](https://my.sqreen.com/profile/organization/integrations).
+2. Add a new "Push API" integration and select `datadog` as the destination.
+3. Choose the corresponding intake server URL. For Datadog US: `http-intake.logs.datadoghq.com`. For Datadog EU: `http-intake.logs.datadoghq.eu`.
+4. In the `dd_api_key` field, provide a [Datadog API key][2] used to stream the Sqreen data as logs.
+
+After a few seconds, the first logs with `source:sqreen` should be flowing in. 
+
+The assets bundled in this integration will be deployed into your Datadog account.
```
Comment (cswatt) [inline sqreen/README.md:38]:
```suggestion
The assets bundled in this integration are deployed into your Datadog account.
```

File: sqreen/README.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+### Troubleshooting
+
+For support, reach out to [support@sqreen.com](mailto:support@sqreen.com).
+
+## Payload Reference Documentation
+
+The log format is fully documented on the [Sqreen documentation](https://docs.sqreen.com/integrations/datadog-integration/).
```
Comment (cswatt) [inline sqreen/README.md:46]:
```suggestion
The log format is fully documented in the [Sqreen documentation](https://docs.sqreen.com/integrations/datadog-integration/).
```

File: sqreen/README.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+
+After a few seconds, the first logs with `source:sqreen` should be flowing in. 
+
+The assets bundled in this integration will be deployed into your Datadog account.
+
+### Troubleshooting
+
+For support, reach out to [support@sqreen.com](mailto:support@sqreen.com).
+
+## Payload Reference Documentation
```
Comment (cswatt) [inline sqreen/README.md:44]:
```suggestion
## Payload reference documentation
```

**Review by cswatt — CHANGES_REQUESTED** [review]
docs style review

File: sqreen/README.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+
+## Overview
+
+The Sqreen integration enables you and your team to monitor your service security activity right from Datadog.
+
+It provides you with dashboards, security rules, log facets, and saved views.
+
+This integration is only available to Datadog and Sqreen design partners. 
+
+It requires the Sqreen microagent to be deployed in the web services to secure.
```
[the remark this answers is not in the document]
Comment (arnaudbreton) [reply sqreen/README.md:11]: the web services

File: sqreen/README.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+3. Choose the corresponding intake server URL. For Datadog US: `http-intake.logs.datadoghq.com`. For Datadog EU: `http-intake.logs.datadoghq.eu`.
+4. In the `dd_api_key` field, provide a [Datadog API key][2] used to stream the Sqreen data as logs.
+
+After a few seconds, the first logs with `source:sqreen` should be flowing in. 
+
+The assets bundled in this integration are deployed into your Datadog account.
+
+### Troubleshooting
+
+For support, reach out to [support@sqreen.com](mailto:support@sqreen.com).
```
Comment (cswatt) [inline sqreen/README.md:42]:
```suggestion
For support, reach out to [Datadog Support](/help).
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**